Description: Fix Django 1.10 compat
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Forwarded: no
Last-Update: 2016-07-19

--- python-django-openstack-auth-2.3.0.orig/openstack_auth/tests/settings.py
+++ python-django-openstack-auth-2.3.0/openstack_auth/tests/settings.py
@@ -63,3 +63,10 @@ POLICY_FILES = {
     'identity': 'keystone_policy.json',
     'compute': 'nova_policy.json'
 }
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'APP_DIRS': True,
+    },
+]
